### ISSUE TITLE
False negative for ERROR lines

### ISSUE THREAD

**Issue by SimoRubi** [OCA/maintainer-quality-tools#610]
Hi,
in https://github.com/OCA/l10n-italy/pull/1465 I'm experiencing a false negative result: all tests of the modules are passed but Travis is :x:.
The problem is caused by an automated test that I'm adding to check a new _unique_ SQL constraint: it raises an Integrity Error that is expected and managed in the TEST but Odoo core also writes an _ERROR_ line in the logs and it is counted as a failed test in the function https://github.com/OCA/maintainer-quality-tools/blob/bae88411197b1adf6e017b6c47ab2d662f881b39/travis/test_server.py#L16.

**TL;DR** all the tests in Odoo are passed but Travis is :x:

Could you please take a look?
Thanks!

**Comment by pedrobaeza** [comment]
You have to use `mute_logger` for avoiding the log line with `ERROR`. There's no other way.

**Comment by SimoRubi** [comment]
Thanks, it worked
